content/ja/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -387,7 +387,7 @@ GAになってからさらなる変更を加えることは現実的ではない
 - `CustomResourceWebhookConversion`: [CustomResourceDefinition](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)から作成されたリソースのWebhookベースの変換を有効にします。
 - `DevicePlugins`: [device-plugins](/docs/concepts/cluster-administration/device-plugins/)によるノードでのリソースプロビジョニングを有効にします。
 - `DryRun`: サーバーサイドでの[dry run](/docs/reference/using-api/api-concepts/#dry-run)リクエストを有効にします。
-- `DynamicAuditing`: [動的監査](/docs/tasks/debug-application-cluster/audit/#dynamic-backend)を有効にします。
+- `DynamicAuditing`: [動的監査](/docs/tasks/debug/debug-cluster/audit/#dynamic-backend)を有効にします。
```

Comment:
Just information.
It's not the target of this PR.
This PR looks ok for me.

`DynamicAuditing` is deprecated on 1.19 and [removed](https://github.com/kubernetes/website/commit/c52731f462f3412fabdbe723151fd1f95bc18127) as of now.

Reply:
Oh, I didn't know that. Thanks for your information!

Reply:
We have a few:sweat_smile: pages needed to be updated.

Reply:
`DynamicAuditing` doesn't exist in the latest English [Feature Gates page](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
In Japanese page, the description is still there, just deleted from the list.
So, whether fix the link or not, this line will be deleted eventually in Japanese, too.
I think delete this line by this PR is also good.

Reply:
I thought it was good to delete this line so I deleted it. Please review again 🙇 

https://github.com/kubernetes/website/pull/39528/commits/516442986aaa57cfaf4c26c4c7f579772438dd0f